src/common/hooks/cities.ts

```diff
@@ -0,0 +1,64 @@
+import { KeycloakInstance } from 'keycloak-js'
+import { useKeycloak } from '@react-keycloak/ssr'
+import { QueryClient, useQuery } from 'react-query'
+
+import { endpoints } from 'service/apiEndpoints'
+import { authQueryFnFactory } from 'service/restRequests'
+import { CityResponse } from 'gql/cities'
+
+// type City = {
+//   id: string
+//   name: string
+//   postalCode: string
+//   countryId: string
+// }
+
+type Country = {
+  id: string
+  name: string
+  countryCode: string
+  cities: []
+}
+
+// export function useCity(id: string) {
+//   return useQuery<CityResponse>(endpoints.city.viewCity(id).url)
+// }
```

Comment:
If we don't need the code please delete it, there's no point in having commented out code in the codebase.